packages/model/languages.ts

```diff
@@ -0,0 +1,3 @@
+
+export const supportedLanguages = ['de', 'fr', 'it', 'rm', 'en']
+export const supportedLanguagesRdf = `(${supportedLanguages.map(lang => `"${lang}"`).join(' ')})`
```

Comment:
It appears that `supportedLanguagesRdf` is used more actually So how about the opposite? Only export RDF/JS literals and map them to their `.value` in the vue component?

```suggestion
import { literal } from '@rdf-esm/data-model'

export const supportedLanguages = [ literal('de'), literal('fr'), literal('it'), literal('rm'), literal('en') ]
```

And if anything, I would not have the export what appears to be hardcoded RDF List in turtle ;)

Reply:
Alright, I don't mind...